examples/nuxt/store/index.js

```diff
@@ -0,0 +1,26 @@
+import { getPerson, getIsServer } from '../telefunc/persons.telefunc'
+
+export const state = () => ({
+  firstName: '',
+  lastName: '',
+  isServer: false,
+})
+
+export const mutations = {
+  set(state, { firstName, lastName, isServer }) {
+    state.firstName = firstName
+    state.lastName = lastName
+    state.isServer = isServer
+  },
+}
+export const actions = {
+  async fetch({ commit }) {
+    const person = await getPerson(0)
+    const isServer = await getIsServer()
+
+    commit('set', {
```

Comment:
How about having two actions? One for `person` and one for `isServer`?

Actually, I'm thiking maybe we can rename `isServer` to `telefunctionWasRunInServer`? (That is a tremendous improvement in code readability: you remember when I was commenting in the next.js PR about `isServer` that it doesn't make sense? That's because I didn't understand your intention. If, back then, I would have read `telefunctionWasRunInServer` instead I would have understood your intention right away.)